cmd/performance-profile-creator/cmd/root.go

```diff
@@ -80,12 +89,24 @@ func getDataFromFlags(cmd *cobra.Command) (*profileCreatorArgs, error) {
 		return nil, fmt.Errorf("Error parsing split-reserved-cpus-across-numa flag: %v", err)
 	}
 	profileName := cmd.Flag("profile-name").Value.String()
+	tmPolicy := cmd.Flag("topology-manager-policy").Value.String()
+	if err != nil {
+		return nil, fmt.Errorf("Error parsing topology-manager-policy flag: %v", err)
```

Comment:
We will need to go over all error messages and remove the capital letters and even the "Error" prefix, but is low priority and not part of this PR.

Reply:
No problem, can fix this.